admin-auth/src/test/kotlin/com/epam/drill/admin/auth/UserRepositoryImplTest.kt

```diff
@@ -125,7 +157,7 @@ class UserRepositoryImplTest {
 
         val users = repository.findAllNotDeleted()
 
-        assertEquals(20 - 5, users.size)
+        assertEquals( 2 + 20 - 5, users.size)//2 default users + 20 inserted - 5 deleted
```

Comment:
I suggest asserting actual absence of blocked users, rather than collection size
```
assertTrue(users.none { it.blocked })
```